components/movementsensor/replay/replay_test.go

```diff
@@ -764,8 +764,8 @@ func TestUnimplementedFunctionAccuracy(t *testing.T) {
 	test.That(t, err, test.ShouldBeNil)
 
 	acc, err := replay.Accuracy(ctx, map[string]interface{}{})
-	test.That(t, err, test.ShouldResemble, movementsensor.ErrMethodUnimplementedAccuracy)
-	test.That(t, acc, test.ShouldBeNil)
+	test.That(t, err, test.ShouldResemble, nil)
```

Comment:
What's the difference between `test.ShouldResemble, nil` and just `test.ShouldBeNil`?